disnake/ext/commands/base_core.py

```diff
@@ -232,6 +234,10 @@ async def invoke(self, inter: ApplicationCommandInteraction, *args, **kwargs) ->
         """
         This method isn't really usable in this class, but it's usable in subclasses.
         """
+        if self.guild_only and inter.guild_id is None:
+            await inter.response.send_message("This command cannot be used in dms", ephemeral=True)
```

Comment:
I think this should have DMs capitalised to be more correct

Reply:
That's true, I'll amend the commit